user-research/contributor-research/Introduction.md

```diff
@@ -0,0 +1,31 @@
+## Abstract:
+Knative is a vital tool in the open-source and cloud-native ecosystem, enabling the development of enterprise-level serverless and event-driven applications. However, its complexity poses challenges for both new contributors and maintainers. Providing guidance consumes valuable maintainer time, resulting in challenges when contributors leave after limited engagement.
```

Comment:
I'm not totally sure about the last two sentences on this line. They seem to state your hypotheses as facts, when we are actually seeking to understand what the challenges are.

Reply:
just made some changes to this part. please review.